tasks.py

```diff
@@ -324,3 +324,10 @@ def _replace(pattern):
     run("git commit -a -m 'Automated update for release v{}'".format(version), echo=True)
     run("git tag v{} -m 'See the release notes for details:\n\nhttps://metallb.universe.tf/release-notes/#version-{}-{}-{}'".format(version, version.major, version.minor, version.patch), echo=True)
     run("git checkout main", echo=True)
+
+
+@task
+def unit(ctx):
+    """Run unit tests"""
+    run("go test -short ./...")
+    run("go test -short -race ./...")
```

Comment:
Couldn't we just run once with `-race`? Is there an advantage to the two separate runs? One downside is longer execution time (given that this task will likely be run very often by devs).

Reply:
I think is fine having it be the same as it is today, `go test` will print any test failures and exit. This should be quite fast (2.7s) and if that works, a more heavy run is done for the `-race`.

Reply:
I honestly did this just because I was keeping the same behavior that existed in CI today.  I didn't have a good explanation for why it was run twice, but your explanation makes sense.  :-)